### ISSUE TITLE
NullReferenceException after plugging in tablet on Gentoo

### ISSUE THREAD

**Issue by ttl41** [OpenTabletDriver/OpenTabletDriver#2742]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### Steps to Reproduce

1. Install using official instructions on Gentoo Linux
2. Run `otd`
3. Plug in XP-Pen Deco Pro Medium

### Expected Behavior

The tablet should be detected and be usable

### Observed Behavior

4. ![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/24609727/b563fe52-5979-42d8-91e8-000c5f26df38)
and tablet is not detected in the GUI

### Tablet Device

XP-Pen Deco Pro Medium

### Diagnostic Information

```json
{
  "App Version": "OpenTabletDriver v0.6.0.4",
  "Build Date": "05/04/2022",
  "Operating System": {
    "Platform": 4,
    "ServicePack": "",
    "Version": "6.1.24.0",
    "VersionString": "Unix 6.1.24.0"
  },
  "Environment Variables": {
    "USER": "beltza",
    "TEMP": null,
    "TMP": null,
    "TMPDIR": null,
    "DISPLAY": ":0",
    "WAYLAND_DISPLAY": null,
    "PWD": "/usr/share/OpenTabletDriver",
    "PATH": "/home/beltzal/Programs/basalt/bsltinstall/bin:/home/beltzal/.local/share/pnpm:/home/beltza/.opam/default/bin:/home/beltza/.scripts:/home/beltza/go/bin:/home/beltzal/Android/Sdk/platform-tools:/home/beltzal/Android/Sdk/platform-tools:/home/beltza/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/lib/llvm/15/bin:/home/beltza/.local/bin:/home/beltzal/projects/flutter/bin"
  },
  "HID Devices": [
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:02.1/0000:04:00.0/usb1/1-6/1-6:1.1/0003:5842:4E61.000C/hidraw/hidraw11",
      "Manufacturer": "X-BOWS",
      "ProductName": "NATURE",
      "SerialNumber": "",
      "FriendlyName": "NATURE",
      "VendorID": 22594,
      "ProductID": 20065,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:02.1/0000:04:00.0/usb1/1-6/1-6:1.0/0003:5842:4E61.000B/hidraw/hidraw10",
      "Manufacturer": "X-BOWS",
      "ProductName": "NATURE",
      "SerialNumber": "",
      "FriendlyName": "NATURE",
      "VendorID": 22594,
      "ProductID": 20065,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:08.1/0000:09:00.4/usb5/5-2/5-2.2/5-2.2:1.3/0003:0BDA:485A.0009/hidraw/hidraw8",
      "Manufacturer": "Generic",
      "ProductName": "USB Audio",
      "SerialNumber": "201405280001",
      "FriendlyName": "USB Audio",
      "VendorID": 3034,
      "ProductID": 18522,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:08.1/0000:09:00.3/usb3/3-2/3-2:1.2/0003:28BD:0904.0005/hidraw/hidraw4",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "11 inch PenTablet",
      "SerialNumber": "0000000000",
      "FriendlyName": "11 inch PenTablet",
      "VendorID": 10429,
      "ProductID": 2308,
      "InputReportLength": 10,
      "OutputReportLength": 10,
      "FeatureReportLength": 0,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:08.1/0000:09:00.3/usb3/3-2/3-2:1.1/0003:28BD:0904.0004/hidraw/hidraw3",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "11 inch PenTablet",
      "SerialNumber": "0000000000",
      "FriendlyName": "11 inch PenTablet",
      "VendorID": 10429,
      "ProductID": 2308,
      "InputReportLength": 10,
      "OutputReportLength": 0,
      "FeatureReportLength": 0,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:08.1/0000:09:00.3/usb3/3-1/3-1:1.2/0003:0C76:161E.0001/hidraw/hidraw0",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "USB PnP Audio Device",
      "SerialNumber": "",
      "FriendlyName": "USB PnP Audio Device",
      "VendorID": 3190,
      "ProductID": 5662,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:08.1/0000:09:00.3/usb3/3-2/3-2:1.0/0003:28BD:0904.0003/hidraw/hidraw2",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "11 inch PenTablet",
      "SerialNumber": "0000000000",
      "FriendlyName": "11 inch PenTablet",
      "VendorID": 10429,
      "ProductID": 2308,
      "InputReportLength": 8,
      "OutputReportLength": 0,
      "FeatureReportLength": 0,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:02.1/0000:04:00.0/usb1/1-5/1-5:1.0/0003:258A:0036.0006/hidraw/hidraw5",
      "Manufacturer": "SINOWEALTH",
      "ProductName": "Wired Gaming Mouse",
      "SerialNumber": "",
      "FriendlyName": "Wired Gaming Mouse",
      "VendorID": 9610,
      "ProductID": 54,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:02.1/0000:04:00.0/usb1/1-5/1-5:1.1/0003:258A:0036.0007/hidraw/hidraw7",
      "Manufacturer": "SINOWEALTH",
      "ProductName": "Wired Gaming Mouse",
      "SerialNumber": "",
      "FriendlyName": "Wired Gaming Mouse",
      "VendorID": 9610,
      "ProductID": 54,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:08.1/0000:09:00.4/usb5/5-2/5-2.3/5-2.3:1.0/0003:03F0:0367.000A/hidraw/hidraw9",
      "Manufacturer": "Quanta",
      "ProductName": "QHMD",
      "SerialNumber": "8CC742Z3C9",
      "FriendlyName": "QHMD",
      "VendorID": 1008,
      "ProductID": 871,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:08.1/0000:09:00.4/usb6/6-2/6-2.1/6-2.1:1.2/0003:045E:0659.0008/hidraw/hidraw6",
      "Manufacturer": "Microsoft",
      "ProductName": "HoloLens Sensors",
      "SerialNumber": "53b06715-38c4-46b0-a4cc-07994066dfca",
      "FriendlyName": "HoloLens Sensors",
      "VendorID": 1118,
      "ProductID": 1625,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    },
    {
      "DevicePath": "/sys/devices/pci0000:00/0000:00:02.1/0000:04:00.0/usb1/1-3/1-3:1.0/0003:048D:5702.0002/hidraw/hidraw1",
      "Manufacturer": "ITE Tech. Inc.",
      "ProductName": "ITE Device",
      "SerialNumber": "",
      "FriendlyName": "ITE Device",
      "VendorID": 1165,
      "ProductID": 22274,
      "InputReportLength": -1,
      "OutputReportLength": -1,
      "FeatureReportLength": -1,
      "CanOpen": true
    }
  ],
  "Console Log": [
    {
      "Time": "2023-05-19T11:18:47.1182953+02:00",
      "Group": "Detect",
      "Message": "Searching for tablets...",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1183562+02:00",
      "Group": "Detect",
      "Message": "Configuration overrides exist in '/home/beltza/.config/OpenTabletDriver/Configurations', overriding built-in configurations.",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1183686+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet '10moon 1060N'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.118383+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Acepen AP 1060'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1183964+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Artisul A1201'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184103+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Artisul AP604'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184251+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Artisul M0610 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184334+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon 1060 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184442+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon M106K Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184533+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon M106K'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184625+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon M10K Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184701+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon M10K'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184779+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon M1220'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.118486+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon M1230'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1184946+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon M6'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185169+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon PD1161'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185255+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon PD1560'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185347+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon PD1561'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185437+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon S56K'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185517+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon S620'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185596+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Gaomon S630'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185674+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Genius G-Pen 560'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185751+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Genius i608x'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1185851+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion 420'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1186026+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion G10T'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1186139+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion GC610'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1186262+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion GT-220 V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1186371+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion GT-221 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1186527+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H1060P'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1186671+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H1161'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1186927+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H420'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187043+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H420X'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.118715+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H430P'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187263+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H580X'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187373+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H610 Pro V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187483+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H610 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187593+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H610X'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187699+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H640P'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187807+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H642'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1187914+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion H950P'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188021+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion HC16'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188128+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion HS610'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188235+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion HS611'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.118834+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion HS64'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188444+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion HS95'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188553+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas 13'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188659+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas 16 (2021)'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188769+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas 16'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188877+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas 20'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1188985+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas 22 Plus'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189092+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas Pro 12'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189201+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas Pro 13 (2.5k)'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189307+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas Pro 13'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189413+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas Pro 16'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189521+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas Pro 20'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189631+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Kamvas Pro 24'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189742+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion New 1060 Plus (2048)'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189852+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion New 1060 Plus'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1189957+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Q11K V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190066+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Q11K'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190178+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion Q620M'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190286+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion WH1409 V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190394+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Huion WH1409'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190501+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Parblo A609'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.119061+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Parblo A610 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190722+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Parblo A640 V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190832+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Parblo Intangbo M'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1190944+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Parblo Intangbo S'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1191054+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Parblo Ninos M'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1191161+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Parblo Ninos S'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.119127+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'RobotPen T9A'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1191378+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'UC-Logic 1060N'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1191485+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'UC-Logic PF1209'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1191593+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'UGEE U1600'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1191699+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'UGTABLET M708'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1191838+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK A15 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.119195+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK A15'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192058+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK A30'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192166+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK A50 V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192273+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK A50'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192378+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK S640 V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192485+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK S640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192591+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'VEIKK VK640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192699+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'ViewSonic Woodpad PF0730'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192805+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTE-430'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1192914+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTE-440'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193021+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTE-450'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193128+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTE-460'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193237+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTE-630'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193346+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTE-640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193455+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTE-650'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193563+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTF-430'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193672+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-460'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193779+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-461'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193887+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-470'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1193993+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-480'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194101+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-490'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194209+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-661'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194316+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-670'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194423+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-680'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194543+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTH-690'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194652+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-4100'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194761+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-4100WL'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194868+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-460'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1194973+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-470'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195081+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-471'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195189+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-472'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195297+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-480'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195405+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-490'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195512+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-6100'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195618+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-6100WL'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195725+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-671'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195833+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-672'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1195945+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-680'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196056+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom CTL-690'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196164+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom DTC-133'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196271+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom ET-0405-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196381+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom ET-0405A-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196486+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom FT-0405-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196593+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom GD-0405-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196701+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom GD-0608-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196807+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom GD-0912-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196904+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom GD-1212-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1196979+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom GD-1218-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197054+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom MTE-450'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197129+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-450'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197205+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-451'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197281+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-460'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197356+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-650'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197455+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-651'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197534+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-660'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.119761+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-850'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197685+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-851'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.119776+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTH-860'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197838+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTK-1240'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197914+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTK-440'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1197989+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTK-450'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198065+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTK-540WL'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198141+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTK-640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198217+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTK-650'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198292+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTK-840'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198368+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTZ-1230'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198445+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTZ-1231W'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198519+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTZ-430'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198595+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTZ-431W'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.119867+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTZ-630'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198746+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTZ-631W'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198823+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom PTZ-930'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.11989+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom XD-0405-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1198976+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom XD-0608-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199053+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom XD-0912-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199129+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom XD-1212-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199206+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom XD-1218-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199282+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XenceLabs Pen Tablet Medium'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199357+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XenceLabs Pen Tablet Small'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199435+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XENX P1-640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199511+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XENX P3-1060'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199587+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XENX X1-640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199661+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Artist 12 (2nd Gen)'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199736+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Artist 12 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199812+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Artist 12'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199886+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Artist 13.3 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1199963+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Artist 15.6 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200038+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Artist 15.6'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200113+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Artist 22HD'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200189+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen CT1060'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200265+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen CT430'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200341+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen CT640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200416+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco 01 V2 (variant 2)'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200491+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco 01 V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200567+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco 01'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200643+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco 02'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200719+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco 03'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200794+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco mini4'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.120087+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco mini7'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1200946+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco Pro Medium'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201022+02:00",
      "Group": "Device",
      "Message": "Initializing device '11 inch PenTablet' /sys/devices/pci0000:00/0000:00:08.1/0000:09:00.3/usb3/3-2/3-2:1.2/0003:28BD:0904.0005/hidraw/hidraw4",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201097+02:00",
      "Group": "Device",
      "Message": "Using report parser type 'OpenTabletDriver.Vendors.XP_Pen.XP_PenTiltReportParser'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201174+02:00",
      "Group": "Device",
      "Message": "Set device output: 02-B0-04",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201248+02:00",
      "Group": "Detect",
      "Message": "Found tablet 'XP-Pen Deco Pro Medium'",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201325+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco Pro Small'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201404+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Innovator 16'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.120148+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star 03'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201555+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star 05 V3'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.120163+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star 06'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201706+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star 06C'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201782+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G430'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201886+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G430S V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1201962+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G430S'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202037+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G540 Pro'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202113+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G540'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.120219+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G640 V2'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202268+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G640'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202345+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G640S'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202422+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G960'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202503+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G960S Plus'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202584+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Star G960S'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202661+02:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'XP-Pen Deco Mini7'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202737+02:00",
      "Group": "NullReferenceException",
      "Message": "Object reference not set to an instance of an object.",
      "StackTrace": "   at OpenTabletDriver.Devices.DeviceReader`1.Main()",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202826+02:00",
      "Group": "IPC",
      "Message": "Connected to a client.",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2023-05-19T11:18:47.1202903+02:00",
      "Group": "Settings",
      "Message": "Presets have been refreshed. Loaded 0 presets.",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    }
  ]
}
```

**Comment by gonX** [comment]
This should be partially fixed once 0.6.3.0 lands in Gentoo repositories, assuming the packaging is updated accordingly.

Since 0.6.3.0 we include configuration files for `modprobe` that prevent the base drivers from loading, though only for the next time the driver is tried loaded, which means that the driver could very well be loaded _before_ the package is installed, however this is not the intention.

The Debian and RedHat package does unload the driver on install, so this seems to be the intended way:
https://github.com/OpenTabletDriver/OpenTabletDriver/blob/7bbe66af957d0c31afb64b0272f2b6ef58f971d4/eng/linux/Generic/postinst#L5-L11

#3017 will track the lack of the module removal on install.
